src/main/java/website/frontrow/board/Mover.java

```diff
@@ -164,6 +166,9 @@ public void tick(Level level)
             addToLog("[MOVER]\tMoved to " + this.location.toString());
         }
         applyGravity();
+        
+        ArtificialIntelligence artificialintelligence = new ArtificialIntelligence(level);
+        artificialintelligence.aiMover();
```

Comment:
Making a new object for this seems a bit,.... ugly. Could maybe do it statically or something?